### ISSUE TITLE
Expand FPS weapon attachment data

### ISSUE THREAD

**Issue by alistair3149** [StarCitizenWiki/API#106]
Weapon attachments apply various modifiers to the equipped weapon.  It would be helpful to include those data:

- [x] `SWeaponZeroingParams` for optics attachments (1180f00)
- [ ] Add type (e.g. laser pointer) for underbarrel attachments
- [x] Add data from `SWeaponModifierComponentParams` for all attachments

**Comment by octfx** [comment]
SWeaponModifierComponentParams now available under the `weapon_modifier` key.
![image](https://github.com/StarCitizenWiki/API/assets/6594492/9e04051d-7419-4917-a960-219229057648)

**Comment by octfx** [comment]
Is the type task still open?

**Comment by alistair3149** [comment]
Yes it is still open. The same information is available under description data but I am not sure if there is a more accurate data source other than relying on localization

**Comment by octfx** [comment]
I've checked the xml files, there does not seem to be a dedicated attribute marking laser pointers as such.  
For now I'd argue that the description data is as good as it gets

**Comment by octfx** [comment]
<img width="1575" height="630" alt="Image" src="https://github.com/user-attachments/assets/29f9c567-5a43-4d95-a179-c06f22816fb5" />

V3 now has expansive data on laser pointer and flashlights 🙂 

Each grouped under a `laserpointer` and `flashlight` key
